andela/pippin-ah-frontend
# [Chore: #162727503] Configure Travis and Coveralls

#### What does this PR do?
This PR configures Travis and Coveralls for the project

#### Description of Task to be completed?
- This story should include the basic setup of jest and Travis.
- A simple test for one of the components should be added.

#### How should this be manually tested?
- When a branch remote branch is pushed, Travis should run tests for that branch
- run `dev:test` to see the current test result for the `LandingPage component test`
